app/scenes/Schedule/index.js

```diff
@@ -15,7 +15,7 @@ import moment from 'moment';
 
 import type { ScheduleTalk } from '../../types';
 
-import Splash from 'react-native-smart-splash-screen';
+// import Splash from 'react-native-smart-splash-screen';
```

Comment:
That component is there just to make sure there's a smooth transition from the app loading to the animation at start. The flow currently is:

- LaunchScreen.storyboard shows the image.
- react-native-smart-splash-screen shows the same image while react native populates the Schedule scene.
- Once the component is mounted, react-native-smart-splash-screen fades out, revealing the 'real' components, which then animate.

If we can do that another way I'm more than happy!